CONTRIBUTING.md

```diff
@@ -1,94 +1,107 @@
+**Step 1:** Open an issue describing the article you wish to write and how it
 relates to existing content. Get approval to write your article.
 
 **Step 2:** Fork the `/dotnet/core-docs` repo.
 
 **Step 3:** Create a `branch` for your article.
 
-**Step 4:** Write your article, placing the article in its own folder and any
-needed images in a _static folder located in the same folder as the article.
-Be sure to follow the proper Markdown syntax. If you have code samples,
-place them in a folder within the `/samples/` folder.  For writing guidelines, see
-[our style guide](/styleguide.md). Also do not forget to remove the "wrench" icon 
-(🔧) from the TOC and the file heading, if applicable. 
+**Step 4:** Write your article. You can use [our template file](template.md) as your starting point. It contains our writing guidelines and also explains about the metadata required for each article, such as author information.
+Also read our [voice and tone article](voice-tone.md).
+Place the article in the appropriate folder and any needed images in a media folder located in the same folder as the article.
+Be sure to follow the proper Markdown syntax. 
+If you have code samples, place them in a folder within the `/samples/` folder. 
+ Also do not forget to remove the "wrench" icon (🔧) from the TOC and the file heading, if applicable. 
 
-**Step 5:** Submit a Pull Request from your branch to `dotnet/core-docs/master`.
+**Step 5:** Submit a PR from your branch to `dotnet/core-docs/master`.
 
-**Step 6:** Discuss the Pull Request with the .NET team; make any requested
-updates to your branch. When they are ready to accept the PR, they will add a
-"LGTM" (Looks Good To Me) comment and any other steps that are (maybe) needed.
+**Step 6:** Discuss the PR with the .NET team; make any requested updates to your branch. 
+When we are ready to accept the PR, we will add a "LGTM" (Looks Good To Me) comment and any other steps that are perhaps necessary.
 
-## Guidance checklist ##
+## DOs and DON'Ts ##
 
-Below is a short list of guiding rules that you should keep in mind when you are
-contributing to .NET Core documentation.
+Below is a short list of guiding rules that you should keep in mind when you are contributing to .NET documentation.
 
-- Don't forget to submit an issue before starting work on an article
-- Don't forget to create a separate branch before working on your article
-- Don't update or `merge` your branch after you submit your pull request
-- If updating code samples in `/samples/`, be sure any line number references
-	in your article remain correct
+- **DON'T** surprise us with big pull requests. Instead, file an issue and start a discussion so we can agree on a direction before you invest a large amount of time.
+- **DO** read our [voice and tone](voice-tone.md) guidelines.
+- **DO** use our [template](template.md) file as the starting point of your work. 
+- **DO** create a separate branch before working on your article. 
```

Comment:
Would it be clearer to say that we use GitHub Flow, and point contributors here: https://guides.github.com/introduction/flow/

I tried re-working the "create a separate branch" and "don't merge or update" as a short bullet list, and I couldn't coming up with good, concise wording.